presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -222,6 +265,21 @@ class EventViewModel @Inject constructor(
             is EventScreenEvent.OnDeleteEvent -> {
                 deleteEvent(eventScreenEvent.eventId)
             }
+            is EventScreenEvent.OnVisitorFilterTypeChanged -> {
+                _eventScreenState.update {  eventScreenState ->
+                    eventScreenState.copy(
+                        selectedVisitorFilterType = eventScreenEvent.visitorFilterType)
+                }
+            }
+            EventScreenEvent.LoadVisitors -> {
+                _eventScreenState.update { eventScreenState ->
+                    eventScreenState.copy(
+                        filteredVisitorsGoing = eventScreenState.attendees.filter { it.isGoing },
+                        filteredVisitorsNotGoing = eventScreenState.attendees.filter { !it.isGoing }
```

Comment:
You called this a couple of times

Maybe have a method and reuse it

Reply:
updated